builder/triton/driver_triton.go

```diff
@@ -28,6 +30,36 @@ func NewDriverTriton(ui packer.Ui, config Config) (Driver, error) {
 	}, nil
 }
 
+func (d *driverTriton) GetImage(config Config) (string, error) {
+	computeClient, _ := d.client.Compute()
+	images, err := computeClient.Images().List(context.Background(), &compute.ListImagesInput{
```

Comment:
Is there a cancellation `context` available in the `computeClient` that we could copy a `context` from?  If not, don't worry about it.

Reply:
There isn't one today but that's a fair point. Might be something to consider over in `triton-go` for the future.